src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -151,10 +157,11 @@ public override Task OnDisconnectedAsync(HubConnectionContext connection)
             return Task.WhenAll(tasks);
         }
 
-        public override Task SendAllAsync(string methodName, object[] args)
+        public override async Task SendAllAsync(string methodName, object[] args)
         {
+            await EnsureRedisServerConnection();
```

Comment:
This should be in publish and subscribe, I don't think it needs to be anywhere else. Those are the 2 redis operations we do today.

Reply:
Yeah, got a little over zealous in adding this